### DISCUSSION TITLE
Newest release is not flagged as "latest".

### DISCUSSION THREAD

**Discussion by jeanlain** [community/community#140265] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hi, I can't find a way to flag the most recent release of my app as "latest". There apparently used to be a checkbox for that (which has been removed). The chatbot said the "latest" should be deduced from the release tags, which are formatted properly.

Here are the releases:
https://github.com/jeanlain/STRyper/releases

Thanks for the suggestions.

Jean

**Comment by ucakbeyza** [comment, marked as answer]
To understand the cause of the problem, create a new version with the appropriate version number and label, and make sure it is not a preliminary version. If the problem is not resolved, re-save the version you created to try it, with or without changes

**Reply by jeanlain** [reply]
Thanks. That was due to the "pre-release" setting.
